.travis.yml

```diff
@@ -14,6 +14,7 @@ before_install:
     - pip install git+https://github.com/SciLifeLab/flowcell_parser.git@master
     - pip install git+https://github.com/SciLifeLab/TACA.git@master
     - pip install codecov
+    - pip install inflect==3.0.2
```

Comment:
Is this a requirement in the code somewhere? Should be in `requirements.txt` if so and not the travis config.

Reply:
Also, would love to know where / how this is used. Never heard of this library before but looks super useful!

Reply:
Its a requirement for ngi-pipeline which is a requirement for taca-ngi-pipeline. So it would go in the requirements.txt in ngi-pipeline instead.

Reply:
Ok yeah I see it [here](https://github.com/NationalGenomicsInfrastructure/ngi_pipeline/blob/e4d2d19942c67b213f25dca5bb3331f30b4a6b1f/requirements.txt#L4).

If this is for an upstream dependency it shouldn't be in this package. By importing `ngi_pipeline` we should automatically be requiring this package, right? If it's that a pinned version is required we should pin that in the upstream package instead of here.

Or am I missing something?

Reply:
Moved the inflect pinning to ngi-pipeline!

Reply:
Nice 🚀